core/storage/trie/impl/ephemeral_trie_batch_impl.cpp

```diff
@@ -59,4 +59,14 @@ namespace kagome::storage::trie {
   outcome::result<void> EphemeralTrieBatchImpl::remove(const Buffer &key) {
     return trie_->remove(key);
   }
+
+  outcome::result<RootHash> EphemeralTrieBatchImpl::hash() {
```

Comment:
I think it will be more conventional to our naming to call it 'calculateHash' or something like this.